Bug: Can't run cli

### Versions
- Node: 20.9.0
- OS: Linux Mint 21.2 Cinnamon
- typeorm-extension: 3.1.1

### Reproduction

According to [this guide](https://github.com/tada5hi/typeorm-extension/blob/master/docs/guide/cli.md) I run the command:
`ts-node ./node_modules/typeorm-extension/dist/cli/index.js seed`
and obtain the error:
`Error: Cannot find module './index.js'`

In node_modules by path `./node_modules/typeorm-extension/dist/cli/index.js` file index.js doesn't exists. Only file index.d.ts.

Best regards, Aleksandr.

--- a/docs/guide/cli.md
+++ b/docs/guide/cli.md
@@ -1,35 +1,36 @@
 # CLI
 
+If you use esm, the executable must be changed from `typeorm-extension` to `typeorm-extension-esm`.
 The following commands are available in the terminal:
 - `typeorm-extension db:create` to create the database
 - `typeorm-extension db:drop` to drop the database
-- `typeorm-extension seed` seed the database
+- `typeorm-extension seed:run` seed the database
+- `typeorm-extension seed:create` to create a new seeder
 
-Alternatively, the full command path can be set in the package.json file to run it e.g. with ts-node.
+If the application has not yet been built or is to be tested with ts-node, the commands can be adapted as follows:
 
 ```
 "scripts": {
-  ...
-  "db:create": "ts-node ./node_modules/typeorm-extension/dist/cli/index.js db:create",
-  "db:drop": "ts-node ./node_modules/typeorm-extension/dist/cli/index.js db:drop",
-  "seed": "ts-node ./node_modules/typeorm-extension/dist/cli/index.js seed"
-  ...
+    "db:create": "ts-node ./node_modules/typeorm-extension/bin/cli.cjs db:create",
+    "db:drop": "ts-node ./node_modules/typeorm-extension/bin/cli.cjs db:drop",
+    "seed:run": "ts-node ./node_modules/typeorm-extension/bin/cli.cjs seed:run",
+    "seed:create": "ts-node ./node_modules/typeorm-extension/bin/cli.cjs seed:create"
 }
 ```
+To test the application in the context of an esm project, the following adjustments must be made:
+- executable `ts-node` to `ts-node-esm`
+- library path `cli.cjs` to `cli.mjs`
 
-:::info Hint
-Read the [Seeding Configuration](seeding#configuration) section to find out how to specify the path,
+Read the [Seeding Configuration](./seeding#configuration) section to find out how to specify the path,
 for the seeder- & factory-location.
-:::
 
-## Options
+#### CLI Options
 
-| Option                  | Commands                        | Default         | Deprecated | Description                                                                                                                                                                                                                              |
-|-------------------------|---------------------------------|-----------------|------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `--root` or `-r`        | `db:create`, `db:drop` & `seed` | `process.cwd()` | `no`       | Path to the data-source / config file.                                                                                                                                                                                                   |
-| `--dataSource` or `-d`  | `db:create`, `db:drop` & `seed` | `data-source`   | `no`       | Name of the data-source file.                                                                                                                                                                                                            |
-| `--synchronize` or `-s` | `db:create` & `db:drop`         | `yes`           | `no`       | Synchronize the database schema after database creation. Options: `yes` or `no`.                                                                                                                                                         |
-| `--initialDatabase`     | `db:create`                     | `undefined`     | `no`       | Specify the initial database to connect to. This option is only relevant for the `postgres` driver, which must always to connect to a database. If no database is provided, the database name will be equal to the connection user name. |
-| `--seed`                | `seed`                          | `undefined`     | `no`       | Specify a specific seed class to run.                                                                                                                                                                                                    |
-| `--connection` or `-c`  | `db:create`, `db:drop` & `seed` | `default`       | `yes`      | Name of the connection. Required if there are multiple connections.                                                                                                                                                                      |
-| `--config` or `-f`      | `db:create`, `db:drop` & `seed` | `ormconfig.js`  | `yes`      | Name to the config file.                                                                                                                                                                                                                 |
+| Option                  | Commands                                           | Default         | Description                                                                                                                                                                                                                              |
+|-------------------------|----------------------------------------------------|-----------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `--root` or `-r`        | `db:create`, `db:drop`, `seed:create` & `seed:run` | `process.cwd()` | Root directory of the project.                                                                                                                                                                                                           |
+| `--dataSource` or `-d`  | `db:create`, `db:drop` & `seed:run`                | `data-source`   | Name (or relative path incl. name) of the data-source file.                                                                                                                                                                              |
+| `--synchronize` or `-s` | `db:create` & `db:drop`                            | `yes`           | Synchronize the database schema after database creation. Options: `yes` or `no`.                                                                                                                                                         |
+| `--initialDatabase`     | `db:create`                                        | `undefined`     | Specify the initial database to connect to. This option is only relevant for the `postgres` driver, which must always to connect to a database. If no database is provided, the database name will be equal to the connection user name. |
+| `--name`                | `seed:create` & `seed:run`                         | `undefined`     | Name (or relative path incl. name) of the seeder.                                                                                                                                                                                        |
+| `--preserveFilePaths`   | `db:create`, `db:drop`, `seed:create` & `seed:run` | `false`         | This option indicates if file paths should be preserved and treated as if the just-in-time compilation environment is detected.                                                                                                          |

